### PR TITLE
Imp/webapp style

### PR DESCRIPTION
**Pull request by rlebre**
Updated webapp's color scheme:

screen shot 2018-07-09 at 17 58 03
screen shot 2018-07-09 at 17 57 49
screen shot 2018-07-09 at 17 57 40
screen shot 2018-07-09 at 17 55 22

Also updated basic styles.
**Before:**
screen shot 2018-07-09 at 18 06 57
screen shot 2018-07-09 at 18 06 49

**Now:**
screen shot 2018-07-09 at 18 04 59
screen shot 2018-07-09 at 18 04 49

### CODE REVIEW

File: dicoogle/src/main/resources/webapp/sass/modules/_management.scss [hunk 1]
```diff
@@ -31,6 +31,7 @@
       color: $blue1;
       &:hover {
         background: $hover_color;
+        color: #ffffff;
```
Comment (bastiao) [inline dicoogle/src/main/resources/webapp/sass/modules/_management.scss]: can we also use variable here and also in sequel? :)

Butt very nice piece of work! :)

Comment (rlebre) [reply dicoogle/src/main/resources/webapp/sass/modules/_management.scss]: We can indeed. Last commits replace that case and other cases of button color with a variable :)

**Comment by rlebre** [comment]
screen shot 2018-07-10 at 16 11 40
screen shot 2018-07-10 at 16 11 07

**Comment by bastiao** [comment]
LGTM. 👍 
Very nice initiative!

**Pull request merged**